src/core/rules.js

```diff
@@ -506,6 +566,57 @@ class Rules extends Rule {
         return this;
     }
 
+    shouldExcludeOn(type, feature) {
+        if (this.excludeOn === null) {
+            return false;
+        }
+
+        for (let entry of this.__excludeOn) {
+            if (entry.type !== '*' && entry.type !== type) {
+                // Check for equal types, if type is not a wildcard
+                continue;
+            }
+
+            if (entry.type === '*' && entry.feature === '*') {
```

Comment:
More (higher-level) comments would be more helpful (e.g. what is the current case/state and how it will be solved)

Reply:
I will add more comments to the branches